### ISSUE TITLE
[question] Ignore system package manager checks if a package is installed manually?

### ISSUE THREAD

**Issue by wbehrens-on-gh** [conan-io/conan#13342]
### What is your question?

I have a yocto toolchain where udev was installed manually. Whenever I try and build something that depends on libudev though I'm given an error 

`libudev/system: ERROR: while executing system_requirements(): System requirements: 'libudev-dev' are missing but can't install because tools.system.package_manager:mode is 'check'.Please update packages manually or set 'tools.system.package_manager:mode' to 'install' in the [conf] section of the profile, or in the command line using '-c tools.system.package_manager:mode=install'`

Basically dpkg-query can't find udev, I'm assuming because it wasn't installed with dpkg, despite the package being available in the toolchain. Is there a way to disable package manager checks in a profile or for specific packages?

### Have you read the CONTRIBUTING guide?

- [X] I've read the CONTRIBUTING guide

**Comment by czoido** [comment]
Hi @WilliamBehrens,
Thanks for your question, I think you could set the `tools.system.package_manager:mode=report` and no errors would be raised. You can check that config in the docs: https://docs.conan.io/2/reference/tools/system/package_manager.html#configuration-properties-that-affect-how-system-package-managers-are-invoked
Hope this helps!
